child_compassion/views/child_compassion_view.xml

```diff
@@ -239,6 +239,14 @@
                         <group attrs="{'invisible': [('type', '!=', 'Transition')]}"/>
                         <group attrs="{'invisible': [('type', '!=', 'Registration')]}">
                         </group>
+                        <group>
+                            <field name="current_project"/>
+                            <field name="new_project"/>
+                            <field name="new_program"/>
+                            <field name="previously_active_program"/>
+                            <field name="status"/>

```

Comment:
You can move the status field after the comments field (at the first group).
You can move other new fields under transfer groups (have same number of fields between the two groups)
You can move field death_intervention under the unplanned exit group